train_hierarchialXC.py

```diff
@@ -41,6 +46,29 @@
 
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Train your model ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-my_hierarchical_XC.train(loader, multiclass.OneVsRestClassifier, estimator=svm.SVC())
+my_hierarchical_XC.train(loader, multiclass.OneVsRestClassifier, estimator=svm.SVC(gamma="scale"))
 predictions = my_hierarchical_XC.predict(loader.get_data()[0])
-print(predictions.toarray())
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Calculate the metrics ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+pred_y = []
+actual_y = []
+for x, y in iter(loader):
+    pred_y.append(predictions.toarray())
+    actual_y.append(y.numpy().reshape(-1))
+
+print("Predicted Y values :: Shape : {} , Values: {}", np.array(pred_y).shape, pred_y)
```

Comment:
for 50 data points
pred_y has shape (50, 50, 159)
actual_y has shape (50, 1, 159)
This is giving erros in calculating the metrics.

Reply:
Why are you appending `predictions` `len(loader)` number of times?

Reply:
Aaah thanks for pointing that out
I'll fix it

Reply:
Lol